react/ProductSummaryBuyButton.tsx

```diff
@@ -1,44 +1,60 @@
 import React from 'react'
-import PropTypes from 'prop-types'
-import BuyButton from 'vtex.store-components/BuyButton'
-import { withRuntimeContext } from 'vtex.render-runtime'
+import { BuyButton } from 'vtex.store-components'
```

Comment:
This component still uses `BuyButton`, but we deprecated it on `store-components`. Should we refactor ProductSummaryBuyButton to use `add-to-cart-button` in the future?

Reply:
That's the pair, we should deprecate this component too.